File: CodeForces/Codeforces Round 871 (Div. 4)/b.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    int t;
    cin >> t;

    while (t--) {
        int n;
        cin >> n;

        vector<int> a(n);
        for (int i = 0; i < n; i++) {
            cin >> a[i];
        }

        int max_len = 0, curr_len = 0;
        for (int i = 0; i < n; i++) {
            if (a[i] == 0) {
                curr_len++;
            } else {
                max_len = max(max_len, curr_len);
                curr_len = 0;
            }
        }

        max_len = max(max_len, curr_len);
        cout << max_len << endl;
    }

    return 0;
}

File: CodeForces/Codeforces Round 871 (Div. 4)/b.py
t = int(input())   # number of test cases

for i in range(t):
    n = int(input())   # length of the array
    a = list(map(int, input().split()))   # elements of the array
    
    max_len = 0   # variable to store the maximum length of blank space found so far
    curr_len = 0   # variable to store the current length of blank space
    
    for j in range(n):
        if a[j] == 0:   # if the current element is 0
            curr_len += 1   # increment the current length of blank space
        else:   # if the current element is 1
            max_len = max(max_len, curr_len)   # update the maximum length of blank space found so far
            curr_len = 0   # reset the current length of blank space
    
    max_len = max(max_len, curr_len)   # update the maximum length of blank space found so far
    
    print(max_len)   # print the maximum length of blank space for the current test case
